src/components/Editor/Project/Project.js

```diff
@@ -95,14 +97,39 @@ const Project = (props) => {
     return () => clearTimeout(debouncer)
   }, [dispatch, forWebComponent, project, user])
 
+  const [params, containerRef] = useContainerQuery(projContainer);
+  const [defaultWidth, setDefaultWidth] = useState('auto');
+  const [defaultHeight, setDefaultHeight] = useState('auto');
+  const [maxWidth, setMaxWidth] = useState('100%');
+  const [handleDirection, setHandleDirection] = useState('right');
+
+  useMemo(() => {
+    const isDesktop = params['width-larger-than-880'];
+
+    setDefaultWidth(isDesktop ? '50%' : '100%');
+    setDefaultHeight(isDesktop ? '100%' : '50%');
+    setMaxWidth(isDesktop ? '75%' : '100%');
+    setHandleDirection(isDesktop ? 'right' : 'bottom');
```

Comment:
@maxelkins I imagine it's these that'll need playing with to get the balance right 👍